armi/materials/material.py

```diff
@@ -214,19 +215,19 @@ def linearExpansionFactor(self, Tc: float, T0: float) -> float:
         --------
         linearExpansionPercent
         """
-        dLLhot = self.linearExpansionPercent(Tc=Tc)
-        dLLcold = self.linearExpansionPercent(Tc=T0)
+        dLLhot = self.linearExpansionPercent(Tc=Tc) / 100.0
+        dLLcold = self.linearExpansionPercent(Tc=T0) / 100.0
 
-        return (dLLhot - dLLcold) / (100.0 + dLLcold)
+        return exp(dLLhot) / exp(dLLcold)
```

Comment:
Also is there a reason that we removed the "-1" so that this is centered around zero? I don't really care either way, but it's certainly less impactful to downstream code (and would remove a lot of the changes in this PR) if the definition of what was being returned as kept consistent.

Reply:
Yeah that's a good perspective on minimizing the the impacts on downstream code. 

Looking back I suppose I ended up taking it out as I thought it simplified things by removing the need for `+ 1`s all over the place. 

Minimizing impacts on downstream code is probably more important. I'll put it back.